src/AccordionPanelGroup.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import cx from 'classnames';
+
+export const ACCORDIONPANELGROUP_CLASS = 'accordionPanelGroup';
+/**
+ * @module AccordionPanelGroup
+ */
+class AccordionPanelGroup extends React.Component {
+	constructor(props) {
+		super(props);
+		this.accordionPanels = this.props.accordionPanels.map((accordionPanel, i) => {
+			return this.clonePanels(accordionPanel, i, accordionPanel.props.isOpen);
```

Comment:
since `clonePanels` takes a single panel as an argument, could you name it `clonePanel`?

Also, since the third argument is always a property of the first argument, you don't need to pass it in separately, which means that you can simplify this to

```js
this.props.accordionPanels.map(this.clonePanel);
```

However, you should `bind` `this.closePanel` _before_ calling it in this way.